src/pages/CamerasPage/Footer/Information/Information.tsx

```diff
@@ -0,0 +1,24 @@
+import styles from "./Information.module.scss";
+
+type Props = {
+    className: string;
+};
+
+export const Information = () => {
+    //TODO: Add the information with the useSelector
+    return (
+        <div className={styles.informatioWrapper}>
+            <div className={styles.title}>Information</div>
+            <div className={styles.measurementLine}>
+                <div>Position:</div>
+                <div>17</div>
```

Comment:
Put position, 17 and m in the same div unless you want to style them differently